### PR TITLE
Unsaved reminder cues - closes Issue #13

### LINKED ISSUE #13: Visual Cues on Unsaved Reminders
User sees visual cue that reminder has not been saved.

* When a reminder has changes that have not been saved to the database/server, the user should see some kind of visual indication in the sidebar of the application on that particular note.

### PR DESCRIPTION
**Pull request by sljohnson32**
@Tman22 @martensonbj 
Closes #13 

## Purpose

Added a visual cue to the reminders list to illustrate when there is dirty data (aka 'tay-tay' data)

## Approach

We added a conditional within the class of the individual reminder elements based on whether or not there is 'tay-tay' data present.

### Test coverage 

We wrote tests to confirm that the class name is toggling correctly when there is and is not 'tay-tay' data.

### Follow-up tasks

#14 (already DONE!)

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -41,10 +41,26 @@ a {
...
 
 .btn:hover {
   cursor: pointer;
 }
+
+.reminder-status {
+  color: green;
+}
+
+.tay-tay {
```
Comment (Tman22) [inline app/styles/app.css:64]: Not sure how I feel about this.

File: tests/acceptance/new-reminder-test.js [hunk 2]
```diff
@@ -27,12 +27,8 @@ test('should add a new reminder', function(assert) {
     andThen(function() {
       assert.equal(currentURL(), 'reminders/new');
       assert.equal(find('.spec-reminder-title').length, 1, 'should return a title field for new reminder')
-      assert.equal(find('.spec-reminder-date').length, 1, 'should return a date field for new reminder')
-      assert.equal(find('.spec-reminder-notes').length, 1, 'should return a notes field for new reminder')
 
       assert.equal(find('.spec-reminder-title').text(), 'Tell black bear his fly is unzipped', 'should show correct title for new reminder')
-      assert.equal(find('.spec-reminder-date').text(), '2017-02-10', 'should show correct date for new reminder')
-      assert.equal(find('.spec-reminder-notes').text(), 'He is embarrassing himself infront of everyone', 'should show correct notes for new reminder')
```
Comment (Tman22) [inline tests/acceptance/new-reminder-test.js:35]: Why did you remove these? Also whats all the shade for?

File: app/styles/app.css (hunk 1 above)
Comment (sljohnson32) [reply app/styles/app.css:64]: @Tman22 just go with it.  It's the easiest way for @mlimberg and I to remember what the class name means...

File: tests/acceptance/new-reminder-test.js (hunk 2 above)
Comment (sljohnson32) [reply tests/acceptance/new-reminder-test.js:35]: @Tman22 We removed them for formatting reasons.  We only want to be displaying the title now in the left hand column for each reminder.

The shade is our way of giving you praise and honor.
